protected/widgets/menu/assets/menu.css

```diff
@@ -23,61 +14,73 @@
     padding-left: 2em;
 }
 
-
 .menu ul {
     margin: 0;
 }
 
 .menu {
-    overflow: hidden;
-    --animation-duration: 500ms;
```

Comment:
This variable is needed, why was it removed?

Reply:
I've put it back